src/pages/yoonhee/Main/Feeds.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Feed from './Feed';
+
+class Feeds extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      feeds: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/feed.json')
```

Comment:
- method 잘 생략해주셨네요! 👍

- 추가적으로, `http://localhost:3000` 부분도 생략할 수 있습니다. 포트 번호가 바뀔 때마다 에러가 발생하고 그때그때 수정해줘야 하는 번거로움이 있기 때문에, 다음과 같이 생략해서 사용해주세요!
```suggestion
    fetch('/data/feed.json')
```

Reply:
수정했습니다-! 포트번호는 생략으루! 팁 감사합니당!